src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -19,7 +20,7 @@ public static string FrameworkDescription
         {
             get
             {
-                return FrameworkName;
+                return string.Format("{0} {1}", FrameworkName, typeof(object).GetTypeInfo().Assembly.GetName().Version);
```

Comment:
How about a C# code file that is generated by the build with something like:

const string BuildNumber = "<generated Build Number>";

And that's what gets used for this value on .Net Native.

When on official builds <generated Build Number> gets the official build number. When on any other build we put a "sentinel" value like 0.1 or 0.99 etc to mean it is a private/non-official build. (Or use whatever we use for privately built package numbers now).